src/Tribe/Main.php

```diff
@@ -3743,8 +3743,10 @@ public function get_event_link( $post, $mode = 'next', $anchor = false ) {
 					$anchor = preg_replace( '|%title%|', $title, $anchor );
 				}
 
-				return apply_filters( 'tribe_events_get_event_link', '<a href="' . esc_url( tribe_get_event_link( $results ) ) . '">' . $anchor . '</a>' );
+				$link = '<a href="' . esc_url( tribe_get_event_link( $results ) ) . '">' . $anchor . '</a>';
 			}
+
+			return apply_filters( 'tribe_events_get_event_link', $link );
```

Comment:
Here we could pass the `$args` and `$post` variables to allow a better filtering.